src/app/not-found.tsx

```diff
@@ -0,0 +1,14 @@
+import Link from 'next/link';
+
+export default function NotFound() {
+  return (
```

Comment:
Does it need to be es5 function with explicit return?

Reply:
😆 you are right